src/vmm/src/device_manager/mmio.rs

```diff
@@ -504,7 +514,7 @@ impl DeviceInfoForFDT for MMIODeviceInfo {
         self.addr
     }
     fn irq(&self) -> u32 {
-        self.irqs[0]
+        self.irqs.unwrap();
```

Comment:
```suggestion
        self.irq.unwrap()
```

Reply:
All the issues should have been resolved.